packages/fxa-settings/src/components/Ready/index.stories.tsx

```diff
@@ -40,7 +40,7 @@ const ReadyWithLayout = ({
 };
 
 export const SigninConfirmedOrSigninVerified = () => (
-  <ReadyWithLayout viewName="signin-confirmed" />
+  <ReadyWithLayout viewName="signin-confirmed" isSignedIn={true} />
```

Comment:
I think you can drop the `={true}` in truthy boolean props (and if so, ditto for the 14 other instances of `={true}` below).

Reply:
Ah that's true! Thank you!